### DISCUSSION TITLE
云米AI电热水器 Air 双胆优享版 没有热水器开关

### DISCUSSION THREAD

**Discussion by Demon-Ghost** [XiaoMi/ha_xiaomi_home#519] [Ideas]
云米AI电热水器 Air 双胆优享版 没有热水器开关

**Comment by SusanPhevos** [comment]
热水器实体详情页选择任意一个非 off 的模式，能够打开热水器。
<img width="579" height="745" alt="Screenshot from 2025-11-28 15-46-08" src="https://github.com/user-attachments/assets/f30b298f-701b-440b-9564-dfb8fd403764" />
